extension/chrome/settings/modules/contacts.ts

```diff
@@ -115,7 +117,27 @@ Catch.try(async () => {
         await Ui.modal.warning('Could not find any new public keys');
       } else {
         $('#bulk_import #processed').html(replacedHtmlSafe).css('display', 'block'); // xss-safe-factory
-        $('#bulk_import .input_pubkey, #bulk_import .action_process').css('display', 'none');
+        $('#bulk_import .input_pubkey, #bulk_import .action_process, #file_import .action_binary_keyfile').css('display', 'none');
+      }
+    }));
+
+    $('#file_import .action_binary_keyfile').off().click(Ui.event.prevent('double', async target => {
+      $('#binary_file').click();
```

Comment:
Please add comment in the code to add a note about what this does / why is it useful.

Reply:
(eg why is a click on the other element needed, as opposed to just having one element. Eg is the other one hidden for visual reasons? etc)

Reply:
No problem, I wasn't really thinking. For some reason I had it in my head that the input element would provide text input element. I forgot that it is just a button.

Reply:
Actually I am finding applying the button styles to the input element really very uncooperative. So I am hiding the file input and triggering a click on the file input element when the styled button is clicked. Then I have an event handler onchange for the input element.

Reply:
No problem there, just when you do something that doesn't have a clear purpose from the code, it's good to add a comment to explain the intent. I do see that you have since replaced it with code similar to what we do elsewhere, presumably resolving the issues.